static/js/settings_account.js

```diff
@@ -76,17 +76,15 @@ exports.update_avatar_change_display = function () {
 };
 
 function display_avatar_upload_complete() {
-    $('#user-avatar-background').css({display: 'none'});
-    $('#user-avatar-spinner').css({display: 'none'});
```

Comment:
What's the story with the changes here to `settings_account`?  I'd prefer this change to the user-avatar UI be a separate commit with a clear explanation in the commit message for why it's correct.

Reply:
ok, Tim, I'll create a new commit with clear explication